sdk/include/psdk/ctfutb.idl

```diff
@@ -23,12 +24,40 @@ import "msctf.idl";
 #endif
 #endif
 
+typedef enum __MIDL_ITfLangBarItemBalloon_0001
```

Comment:
Where did you get this auto-generated name? This shouldn't be present in IDL files usually.
Based on https://github.com/tpn/winsdk-10/blob/master/Include/10.0.16299.0/um/ctfutb.idl#L552
it seems it's because the enum is within an interface :thinking:

Reply:
I have got them from Microsoft Learn.
https://learn.microsoft.com/ja-jp/windows/win32/api/ctfutb/ne-ctfutb-tflbballoonstyle

Reply:
Deleted them.

Reply:
Moved enum.
Fixed some mistakes.